content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -96,28 +96,15 @@ Once you install Istio and Kiali, deploy the [Bookinfo](/docs/examples/bookinfo/
         $ watch -n 1 curl -o /dev/null -s -w %{http_code} $GATEWAY_URL/productpage
         {{< /text >}}
 
-1.  To determine the Kiali URL, you use the same `GATEWAY_URL` as the Bookinfo application,
-    only with a different port.
+1.  Open the Kiali UI.
 
-    *   If you are running in an environment that has external load balancers,
-        run this command:
+    *   In Kubernetes environments, execute the following command:
```

Comment:
Since there are no two options, you can merge this with the previous numbered step. See comment.